src/components/EventBox/Box.tsx

```diff
@@ -0,0 +1,97 @@
+import { useRouter } from "next/router";
+import { Disclosure, Transition } from "@headlessui/react";
+
+// Contains type of info stored in our event box.
+type BoxProps = {
+  eventId: string;
+  eventTitle: string;
+  startDate: string;
+  endDate: string;
+  description: string;
+  location: string;
+  handleClick?: () => void;
+};
+
+const Box = ({
```

Comment:
please add a sample repair request (maybe by adding default repair request) if this is to be done in another issue please just resolve this comment